app/src/main/java/com/hover/stax/accounts/AccountDetailFragment.kt

```diff
@@ -155,13 +155,11 @@ class AccountDetailFragment : Fragment(), TransactionHistoryAdapter.SelectListen
                 binding.detailsCard.shortcodeBtn.setOnClickListener { Utils.dial(c.rootCode, requireContext()) }
             }
 
-            transactions.observe(viewLifecycleOwner) {
+            listOfTransactionActionPair.observe(viewLifecycleOwner) {
```

Comment:
You've made changes to the adapter. It now does not support the Pair. Pass the list of `TransactionHistory` instead.

Reply:
Weird that this hasn't changed in Github diffs. Will push again.